python/ray/serve/application.py

```diff
@@ -62,7 +62,7 @@ def add_deployment(self, deployment: Deployment):
                 "new_deployment"
             )
 
-        self._deployments[deployment.name] = deployment
+        self.__setitem__(deployment.name, deployment)
```

Comment:
is there a difference here? calling setitem directly is strange

Reply:
It de-dupes the import path conversion logic.